.gitignore

```diff
@@ -8,6 +8,8 @@
 # Don't ignore catalogues and game system
 !*.cat
 !*.gst
+!*.catz
+!*.gstz
```

Comment:
That's incorrect, as well as commiting those files. It's outlined in the docs: https://github.com/BSData/catalogue-development/wiki/Data-Author-Guide#most-common-mistakes

Reply:
Thanks for the review, fixed with commit d16c6cf58435dcddb6e520be46207524c44187ed